robotiq_s_model_articulated_gazebo_plugins/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotiq_s_model_articulated_gazebo_plugins)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS std_msgs gazebo_plugins actionlib tf image_transport control_msgs trajectory_msgs geometry_msgs sensor_msgs roscpp gazebo_ros robotiq_s_model_articulated_msgs)
+
+find_package(gazebo REQUIRED)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+include_directories(
+  ${PROJECT_SOURCE_DIR}/include
+  ${catkin_INCLUDE_DIRS}
+  ${GAZEBO_INCLUDE_DIRS}
+)
+
+link_directories(
+  ${GAZEBO_LIBRARY_DIRS}
+)
+
+catkin_package(
+   DEPENDS gazebo_plugins gazebo_ros roscpp
+)
+
+
+add_library(RobotiqHandPlugin src/RobotiqHandPlugin.cpp)
+set_target_properties(RobotiqHandPlugin PROPERTIES LINK_FLAGS "${ld_flags}")
+set_target_properties(RobotiqHandPlugin PROPERTIES COMPILE_FLAGS "${cxx_flags}")
+target_link_libraries(RobotiqHandPlugin ${catkin_LIBRARIES})
+add_dependencies(RobotiqHandPlugin robotiq_s_model_articulated_msgs_gencpp ${CATKIN_EXPORTED_TARGETS} robotiq_s_model_articulated_msgs_EXPORTED_TARGETS)
```

Comment:
Isn't `robotiq_s_model_articulated_msgs_gencpp` included in `robotiq_s_model_articulated_msgs_EXPORTED_TARGETS`?